playground/src/modes/validator.critical.scss

```diff
@@ -0,0 +1,64 @@
+@import '_mixins.scss';
+
+@import 'components/atoms/_text.scss';
+@import 'components/atoms/_color.scss';
+
+.validator {
+
+  main {
+    flex-direction: row-reverse;
+    margin-left: 0;
+    margin-right: 8px;
+
+    &:after {
+      bottom: auto;
+      right: auto;
+      left: 0;
+      top: 0;
+      width: 25%;
+      max-width: 175px;
+      height: 60%;
+      clip-path: polygon(100% 0, 0 0, 0 100%);
+      @include gradient-websites;
+    }
+  }
+
+  #source {
+    @media (max-width: 767px) {
+      display: none;
+    }
+  }
+
+  #validation {
+    padding: 55px 20px;
+    overflow-y: auto;
+
+    @media (min-width: 786px) {
+      padding: 55px 5px 55px 55px;
+    }
+
+    @media (min-width: 1000px) {
+      padding: 55px;
+    }
+  }
+
+    h1 {
```

Comment:
nit: formatting? Shouldn't prettier handle this?

Reply:
We didn't configure prettier for SCSS yet but I formatted it anyway 😁